test/features/data_qld_theme.feature

```diff
@@ -85,11 +85,6 @@ Feature: Theme customisations
         Then I should see an element with xpath "//a[contains(@href, '/datastore/dump/') and contains(@href, 'format=json') and contains(string(), 'JSON')]"
         Then I should see an element with xpath "//a[contains(@href, '/datastore/dump/') and contains(@href, 'format=xml') and contains(string(), 'XML')]"
 
-    @Publications
-    Scenario: Homepage 'Publish in the Gazettes' link is correct
```

Comment:
why purge? should it not be moved to the publication_qld_theme.feature instead?

Reply:
It's already in `homepage.feature`